modules/afsmtp/afsmtp.c

```diff
@@ -45,6 +45,7 @@ typedef struct
 {
   gchar *phrase;
   gchar *address;
+  LogTemplate *addr_tmpl;
   afsmtp_rcpt_type_t type;
```

Comment:
One further thought: what happens if template compilation fails in either parameters. We should fail the startup as well.